plugins/archlinux/archlinux.plugin.zsh

```diff
@@ -11,7 +11,7 @@ if [[ -x `which yaourt` ]]; then
   alias yaupg='yaourt -Syu'        # Synchronize with repositories before upgrading packages that are out of date on the local system.
   alias yasu='yaourt --sucre'      # Same as yaupg, but without confirmation
   alias yain='yaourt -S'           # Install specific package(s) from the repositories
-  alias yains='yaourt -U'          # Install specific package not from the repositories but from a file 
+  alias yains='yaourt -U'          # Install specific package not from the repositories but from a file
```

Comment:
Hi There, Is it posible to update yaourt to yay?
And I try the systemctl journalctl and netctl aliases with no success at all!